src/app.ts

```diff
@@ -1,6 +1,71 @@
+type ResultData = {
+  id: string;
+  gender: string;
+  age: number;
+  row: number;
+};
+
+type PagingData = {
+  next?: string;
+  previous?: string;
+};
+
+let previousBtn: HTMLElement | any = document.querySelector("button[data-prevbtn]");
+let nextBtn: HTMLElement | any = document.querySelector("button[data-nextbtn]");
+let pageView: HTMLElement | any = document.querySelector("label[data-pageview]");
+let tableBody: HTMLElement | any = document.querySelector("tbody[data-sink]");
+let currentPage: number = 1;
+
+const paginate = () => {
+  previousBtn.disabled = true;
+  previousBtn.style.color = 'black';
```

Comment:
Codacy has a fix for the issue: [Replace `'black'` with `"black"`](https://app.codacy.com/gh/dekunleroy/pipeline_assessment/pullRequest?prid=9962768)
```suggestion
  previousBtn.style.color = "black";
```